### PR TITLE
Fixed handling errors, first load user employee then other requests.

### PR DESCRIPTION
**Pull request by arcelena**

### CODE REVIEW

File: mobile/src/feeds/home-feeds-screen.tsx [hunk 1]
```diff
@@ -67,7 +69,9 @@ class HomeFeedsScreenImpl extends React.Component<FeedsScreenProps & FeedScreenD
     }
 
     public componentDidMount() {
-        this.props.fetchNewFeeds();
+        if (this.props.user) {
```
Comment (santee) [inline mobile/src/feeds/home-feeds-screen.tsx]: i think not. you need to subscribe to event where you load new users. This way it'll check for user (which might be empty) and won't load anything until you scroll

File: mobile/src/reducers/feeds/feeds.epics.ts [hunk 2]
```diff
@@ -13,9 +13,15 @@ import { Alert } from 'react-native';
 import { retryWhen, map } from 'rxjs/operators';
 import { UnaryFunction } from 'rxjs/interfaces';
 import { handleHttpErrors } from '../errors/errors.epics';
+import { LoadUserEmployeeFinished } from '../user/user.action';
 
 export const pagingPeriodDays = 10;
 
+export const loadUserEmployeeFinishedEpic$ = (action$: ActionsObservable<LoadUserEmployeeFinished>, state: AppState, deps: DependenciesContainer) =>
+    action$.ofType('LOAD-USER-EMPLOYEE-FINISHED')
```
Comment (santee) [inline mobile/src/reducers/feeds/feeds.epics.ts]: ah, ok, you do it here... but why do you have this call in componentDidMount in this case?

File: mobile/src/reducers/organization/organization.epics.ts [hunk 3]
```diff
@@ -61,4 +61,12 @@ export const loadEmployeesForUserDepartmentEpic$ = (action$: ActionsObservable<L
...
 export const loadEmployeesForUserRoomEpic$ = (action$: ActionsObservable<LoadUserEmployeeFinished>) =>
     action$.ofType('LOAD-USER-EMPLOYEE-FINISHED')
-        .map(x => loadEmployeesForRoom(x.employee.roomNumber));
\ No newline at end of file
+        .map(x => loadEmployeesForRoom(x.employee.roomNumber));
+
+export const loadUserEmployeeFinishedEpic$ = (action$: ActionsObservable<LoadUserEmployeeFinished>, state: AppState, deps: DependenciesContainer) =>
+    action$.ofType('LOAD-USER-EMPLOYEE-FINISHED')
+        .map(x => {
+            console.log('LOAD ROOM');
```
Comment (santee) [inline mobile/src/reducers/organization/organization.epics.ts]: whoops

File: mobile/src/reducers/errors/errors.epics.ts [hunk 4]
```diff
@@ -24,7 +24,7 @@ function retryWhenErrorOccured<T>(isForceLogout: boolean = false): UnaryFunction
         return errors.exhaustMap((e: any) => new Promise((resolve, reject) => {
```
Comment (serjKim) [inline mobile/src/reducers/errors/errors.epics.ts]: typings

**Pull request merged**